### ISSUE TITLE
Questions about cell type prediction and dataset

### ISSUE THREAD

**Issue by toycat-I** [vandijklab/cell2sentence-ft#7]
Great job! What I want to ask is about the autoregressive cell type prediction task. Is the model pre trained on the training set of size 39936 mentioned in the article, and then subjected to zero shot on the test set of size 4992? Can I publish a dataset with a scale of 49920 and the split used in this work?

**Comment by sachalevy** [comment]
Hey @toycat-I , thank you for your interest in cell2sentence! Concerning the models trained only on C2S, these are indeed pre-trained on the whole training set (39936 cell sentences). For fine-tuned models (from pre-trained gpt2 base models), we use a prompt fine-tuning setup to train the models on the training set.
We're currently preparing an extended version of the dataset which we'll be sharing soon.

**Comment by toycat-I** [comment]
Thank you! What's more, May I ask if it is possible to release the pre-trained checkpoint of cell2sentence for trial purposes?
